src/pages/manage/movies/MovieForm.tsx

```diff
@@ -167,96 +167,132 @@ const convertSubEventsToTimeTable = (subEvents: SubEvent[] = []) => {
 const nextWeekWednesday = nextWednesday(new Date());
 const formatDate = (date: Date) => format(date, 'dd/MM/yyyy');
 
-const MovieForm = () => {
-  const form = useForm<FormData>({
-    resolver: yupResolver(schema),
-    defaultValues: {
-      timeTable: {
-        data: {},
-        dateStart: formatDate(nextWeekWednesday),
-        dateEnd: formatDate(nextWeekWednesday),
-      },
-      eventTypeAndTheme: {
-        eventType: {
-          id: EventTypes.Film,
-          label: 'Film',
-        },
-      },
+const usePublishEvent = ({ id, onSuccess }) => {
+  const queryClient = useQueryClient();
+
+  const publishMutation = usePublishMutation({
+    onSuccess: () => {
+      queryClient.invalidateQueries(['events', { id }]);
+      onSuccess();
     },
   });
 
-  const {
-    handleSubmit,
-    formState: { errors, dirtyFields },
-    reset,
-    watch,
-    trigger,
-  } = form;
+  return async (date: Date = new Date()) => {
+    if (!id) return;
 
-  const { t } = useTranslation();
-  const router = useRouter();
-  const queryClient = useQueryClient();
+    await publishMutation.mutateAsync({
+      eventId: id,
+      publicationDate: formatDateToISO(date),
+    });
+  };
+};
 
-  const [newEventId, setNewEventId] = useState(
-    (router.query.eventId as string) ?? '',
-  );
+const useAddEvent = ({ onSuccess }) => {
+  const addEventMutation = useAddEventMutation();
+  const changeTypicalAgeRangeMutation = useChangeTypicalAgeRangeMutation();
+  const addLabelMutation = useAddLabelMutation();
+  const createProductionWithEventsMutation = useCreateProductionWithEvents();
+  const addEventToProductionByIdMutation = useAddEventToProductionById();
```

Comment:
```suggestion
  const createProductionWithEventsMutation = useCreateProductionWithEvents();
  const addEventToProductionByIdMutation = useAddEventToProductionById();
```

These 2 aren't renamed to `...Mutation`

Reply:
I think they are aliased on top of the file, but let me check.

Reply:
Resolved it in the first PR, since that one triggered these changes